### DISCUSSION TITLE
What is the name of the music player that appears in the video?

### DISCUSSION THREAD

**Discussion by fmolero69** [prasanthrangan/hyprdots#1553] [Q&A]
What is the name of the music player that appears in the video?
And why is it not integrated into the packages that were installed?

![240529_15h38m22s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/116031640/bc2e334c-4bd1-41f0-8f73-328b234314a9)

**Comment by kRHYME7** [comment]
Spotify + spicetify-cli.

Install both and run ` wallbashspotify.sh --reset `.

**Reply by fmolero69** [reply]
Thank you!
